docs/reference/administration_reference.md

```diff
@@ -0,0 +1,96 @@
+# Administration reference guide
+
+## Overview
+
+This document provides a reference guide for all the attributes available for `invenio-administration` views.
+
+## List View
+
+### Attributes
+| Attribute                     | Required   | Default                                | Description                                                                                                          |
+|-------------------------------|------------|----------------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| **name**                      |            | *View class name*                      | - view's name used by Flask to create routing                                                                        |
+| **resource_config**           | *required* |                                        | used to retrieve the resource (refers to extension attribute - check ext.py)                                         |
+| **search_request_headers**    |            | *{ "Accept" :  "application/json" }    | specifies resource-specific headers to add during REST API request                                                   |
```

Comment:
This happens in some other values (see "outdated" tho not really outdated) comments a bit above in the conversation.

```suggestion
| **search_request_headers**    |            | *{ "Accept" :  "application/json" }*    | specifies resource-specific headers to add during REST API request                                                   |
```